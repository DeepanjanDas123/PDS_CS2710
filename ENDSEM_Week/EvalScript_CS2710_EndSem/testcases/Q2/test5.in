DecodeMessage
((N,(h,P)),(((6,I),(y,X)),(E,(G,c))));
1101010011001001100011101010111010101101101110110111110010001010001111
Quit
