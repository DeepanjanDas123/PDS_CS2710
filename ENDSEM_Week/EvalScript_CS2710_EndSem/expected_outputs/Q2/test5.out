EyPNI6GyGyEEGEcINh6c
